src/create/getUserInput.ts

```diff
@@ -24,6 +24,12 @@ export async function getUserInput() {
           },
         });
       },
+      setupModules: () => {
+        return prompts.confirm({
+          message: 'Install demo apps',
```

Comment:
I don't like the word `demo` or `apps`.

Those are experiences part of one app.

And App can have multiple experiences.

Reply:
any suggestions on the wording?

Reply:
Actually how about `experiences`? It would be nice to create a grouping for the BOAT experiences. Something like `(experiences)/_layout` (containing header, footer, etc) would be useful.

I'd like to make `src/*` mostly to contain app agnostic code and move app specific components closer to `app/(experiences)`.

Reply:
Oh I see, you were never part of the sync meetings we did before. We always called them experiences. :)